spec/features/user_sees_the_profile_page_for_a_bookstore_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+feature "user sees a specific bookstore page" do
+  zip = "18900"
+  let!(:bookends) do
+    Bookstore.create(
+      name: "Bookends",
+      address: "559 Main St",
+      city: "Winchester",
+      state: "MA",
+      zip_code: zip.to_i,
+      url: "http://www.bookendswinchester.com/",
+      img_url: "http://www.booksite.com/cilla/bookends/CAIZ8923.jpg",
+      description: "A nice little bookstore",
+      beverage: false,
+      food: true
+    )
+  end
+  let!(:barnes) do
```

Comment:
add a blank line between the two `let` blocks...they're long so it's hard to read.